cmd/oras/discover.go

```diff
@@ -150,14 +151,24 @@ func fetchAllReferrers(ctx context.Context, repo *remote.Repository, desc ocispe
 	for _, r := range results {
 		// Find all indirect referrers
 		referrerNode := node.AddPath(r.ArtifactType, r.Digest)
+		if opts.Verbose {
+			for k, v := range r.Annotations {
+				v := map[string]string{k: v}
+				bytes, err := yaml.Marshal(v)
+				if err != nil {
+					return err
+				}
+				referrerNode.AddPathString(strings.TrimSpace(string(bytes)))
```

Comment:
What's the reason to trim space?

Reply:
1. yaml.Marshal will add a new line automatically at the end of a object, which makes the tree view less prettified
2. Head/trailing spaces in key/value will be quoted by yaml.Marshal, thus trimming space will cause unwanted change to annotation key/value

Reply:
Sounds good.